src/template/wasm2c.declarations.c

```diff
@@ -1,4 +1,10 @@
 
+#if WABT_BIG_ENDIAN
+#define LEADDR(mem, addr, n) (mem)->data[(mem)->size - (addr) - (n)]
```

Comment:
Can we name this something a little more friendly looking (how do you feel about `MEM_ADDR`), and maybe with a comment about its purpose and the meaning of the arguments?

Reply:
We would insist on keeping it as `LEADDR`. In the hypothesis wasm ever adds a big-endian memory type (and an accompanying `v128be`), we can easily introduce `BEADDR` and have a mix of little-endian and big-endian memories in the same module. :sweat_smile:

Reply:
Please correct me if I'm wrong, but I think there is no proposal about this sort of change at the moment. I think, we can always refactor this to LEADDR, if that happens. For the current wasm support, however, I think @keithw 's suggestion of the MEM_ADDR name will keep the code more readable.

Reply:
Hmm, yeah, I don't want the tail to end up wagging the dog here. We care about big-endian support (at least insofar as we have users who care), but not to the degree that it ends up making the codebase confusing or imposing to new contributors. There's nothing inherently little-endian about an expression like `&(mem)->data[addr]`.

How would you feel about `MEMREF` ? Or anything else you want to propose?